src/components/Modal/modal_two.tsx

```diff
@@ -0,0 +1,24 @@
+import { Modal, ModalOverlay, ModalContent, ModalBody } from "@chakra-ui/react";
```

Comment:
Give this file a good name so that the function of the file can be easily understood without stress.